src/constants.py

```diff
@@ -1 +1 @@
-BASE_DIR = "/Users/wyattshapiro/projects/dog_walk/"
+BASE_DIR = "/Users/williamseife/Documents/GitHub/better-walk/"
```

Comment:
change this to the following:
`import os
BASE_DIR = os.path.dirname(os.path.dirname(os.path.realpath(__file__)))`

This finds the real path of the file (constants.py), gets the directory (src/) and then gets that directory (better-walk/)

Reply:
done.